content/faculty/faculty_dean.md

```diff
@@ -7,19 +7,20 @@ submenu_index: 4
 
 ### Декан
 
->доктор социологических наук, профессор Симхович Валентина Александровна
+>кандидат экономических наук, доцент Петриченко Елена Владимировна
```

Comment:
`>` should be followed by a single whitespace. Please take a look at a [reference on blockquotes in Markdown](https://daringfireball.net/projects/markdown/syntax#blockquote).